src/docs/api/Log.md

```diff
@@ -0,0 +1,28 @@
+# LOG Reference
+
+### Logging Levels
+
+> this.log
+
+Basic logging 
+
+> this.log.info
+
+Basic logging
+
+> this.log.success
```

Comment:
i had a thought maybe to specify this is only in homebridge 1.8.0 and later but then again this was released many. months ago now so maybe not necessary